rust-symcrypt/src/hmac.rs

```diff
@@ -84,6 +110,372 @@ pub trait HmacState: Clone {
     fn result(self) -> Self::Result;
 }
 
+/// `HmacMd5ExpandedKey` is a struct that represents the expanded key for the [`HmacMd5State`].
+#[cfg(feature = "md5")]
+struct HmacMd5ExpandedKey(symcrypt_sys::SYMCRYPT_HMAC_MD5_EXPANDED_KEY);
+// Wrapping the expanded key so that it can be independently dropped after it's ref count has gone to 0
+
+// Since HmacMd5ExpandedKey can be referenced multiple times, HmacMd5ExpandedKey must be ref counted and a there needs to be a separate drop()
```

Comment:
> a [](http://example.com/codeflow?start=104&length=2)

nitpick: extra word

Reply:
Fixed for this one and the other hashing algos.